File: subroutines/all/subroutine leftadjustline.f
      subroutine leftadjustline(line,ic1,ic2)
      character*(*) line
      icol=ic1
      call nextchar(line,icol,132)
      nshift=icol-ic1
      if (nshift .gt. 0) then
        do i=icol,ic2
          line(i-nshift:i-nshift)=line(i:i)
        end do
        do i=1,nshift
          line(ic2-i+1:ic2-i+1)=' '
        end do
      end if
      return
      end

File: fortran to python conversion/dependencies/subroutine leftadjustline.py
def leftadjustline(line, ic1, ic2):
    """

    Adjust line on the left

    ***
    Parameters;
    line:
    ic1:
    ic2:

    Returns:
    line

    """
    icol = ic1
    icol = nextchar(line, icol, 132)
    nshift = icol - ic1
    if (nshift > 0):
        for i in range(icol, ic2):
            line[i-nshift:i-nshift+1] = line[i:i+1]
        for i in range(nshift):
            line[ic2-i+1:ic2-i+2] = " "
    return line
